compass/provenance.py

```diff
@@ -110,27 +101,9 @@ def write(work_dir, test_cases, mpas_core=None, config_filename=None,
     provenance_file.close()
 
 
-def _get_mpas_git_version(mpas_core, config_filename, mpas_model_path):
```

Comment:
It looks like this section was moved to a different file.  This seems somewhat unrelated to the PR description, so just checking this was intentional to include in this PR.  If it's just a convenience thing for planned future changes, that's fine with me.

Reply:
I did pull this out into its own PR: https://github.com/MPAS-Dev/compass/pull/378

Reply:
I was fine with it being included in this PR, but pulling it out is fine too.